content/blog/2023/03/camel-quarkus-release-3.0.0-M1/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Camel Quarkus 3.0.0-M1 Released"
+date: 2023-03-22
+authors: ["jamesnetherton","zbendhiba"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 3.0.0-M1 Released"
+summary: "Camel Quarkus 3.0.0-M1 is the first iteration towards a new 3.0.0 major release"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 3.0.0-M1 release of Camel Quarkus. This is the first iteration towards a new 3.0.0 major release.
+It aligns with Quarkus 3.0.0.Alpha5 and Camel 4.0.0-M2.
+This is the first release of Camel Quarkus with JDK 17 as baseline and upgraded to Jakarta EE 10.
+
+## Camel 4.0.0-M2
+Please refer to the [Camel 4.0.0-M2 announcement](/content/blog/2023/03/RELEASE-4.0.0-M2/) for more details.
+
+## Quarkus 3.0.0.Alpha5
+
+Please refer to the [3.0.0.Alpha5 announcement](https://quarkus.io/blog/quarkus-3-0-0-alpha5-released/) for more details.
+
+# Migrating to Camel Quarkus 3.0.0
+Please check the [Camel Quarku 3.0 migration guide](TODO) to learn about main breaking changes from both Camel 4.0 and Quarkus 3.0. 
```

Comment:
The TODO here refers to publishing a migrating page, if we are confident 
I submitted a PR: https://github.com/apache/camel-quarkus/pull/4680